### ISSUE TITLE
Talkback is not announcing labels for Name, Description, Category edit fields in Project screen: A11y_.NET 9 template_Projects_Labels or Instructions.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#26911]
### GitHub Tags:
#A11yTCS; #A11ySev1; #Android; #WCAG3.3.2; #SH_.NET9template_Android_Dec2024; #Talkback; #.NET Core; #Device; #A11yMAS; #Labels or Instructions; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
Android Version: 14
Talkback version: 15.1

### Repro Steps:

1. Turn on Talkback and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till project section and double tap on any project, Project screen will open,
4. Swipe till category section and choose a category from the list of category.
5. Swipe till tags section and choose a tag from the list of tags or We can select the task from the existing task list.\
6. Swipe till new task button in the project screen and double tap on it.
7. New Project screen gets opened.
8. Swipe till Name, Description, Category edit fields and Observe that Talkback is not announcing labels for Name, Description, Category edit fields in Project screen.

### Actual Result:
Talkback is not announcing labels for Name, Description, Category edit fields in Project screen.
NOTE: Issue is also reproducing in Tasks screen, that Talkback is not announcing completed label along with check box.

### Expected Result:
Talkback should be announcing labels for Name, Description, Category edit fields in Project screen.

### User Impact:
Users who relay on screen reader wont get proper information of Name, Description, Category edit fields.

### Attachment:

https://github.com/user-attachments/assets/b7b80f8a-9f60-46db-852f-f6dd282e0af0

https://github.com/user-attachments/assets/1bf1dcd8-f172-4dbd-94c2-d2c6cce6327b

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by keerthiduvvuri** [comment]
Verified in latest Environment, Issue is Fixed. Hence closing the bug.
Added attachment for reference.
#Regressed:03-06-25; #Closed; #A11yFixed;

https://github.com/user-attachments/assets/94dd5f31-765a-4a26-8db3-b6d95875a8c6

https://github.com/user-attachments/assets/282b1632-eda2-4791-9a42-9ed5cbf71375
